src/main/java/org/commonjava/indy/service/archive/jaxrs/ArchiveManageResources.java

```diff
@@ -138,7 +138,7 @@ public Response create( final @Context UriInfo uriInfo, final @Context HttpReque
 
     @Operation( description = "Get latest historical build archive by buildConfigId" )
     @APIResponse( responseCode = "200", description = "Get the history archive successfully" )
-    @APIResponse( responseCode = "204", description = "The history archive doesn't exist" )
+    @APIResponse( responseCode = "404", description = "The history archive doesn't exist" )
```

Comment:
This is not changing the 404 for api response. It is used for API doc for swagger-ui. I think something need  to check is for the code logic: when file not found, build an error response with 404 status.

Reply:
@ligangty, yep, careless, fix in the new commit: https://github.com/Commonjava/indy-archive-service/pull/10/commits/8844d59c131d66cf2278ff815a77cfeaec760a7a